samples/contrib/azure-samples/kfp-azure-databricks/tests/test_dbfsblock_op.py

```diff
@@ -0,0 +1,196 @@
+import unittest
+from pathlib import Path
+import kfp
+from kfp.dsl import PipelineParam
+from databricks import CreateDbfsBlockOp, DeleteDbfsBlockOp
+
+class TestCreateDbfsBlockOp(unittest.TestCase):
+
+    def test_databricks_createdbfsblock_without_k8s_or_block_name(self):
+        def my_pipeline():
+            CreateDbfsBlockOp(
+                name="createdbfsitem",
+                data="cHJpbnQoImhlbGxvLCB3b3JsZCIpCgoK",
+                path="/Users/alejacma@microsoft.com/ScalaExampleNotebook"
+            )
+
+        self.assertRaises(ValueError, lambda: kfp.compiler.Compiler()._compile(my_pipeline))
+
+    def test_databricks_createdbfsblock(self):
+        def my_pipeline():
+            block_name = "createdbfsitem"
+            data = "cHJpbnQoImhlbGxvLCB3b3JsZCIpCgoK"
+            path = "/Users/alejacma@microsoft.com/ScalaExampleNotebook"
+            
+
+            expected_spec = {
+                "path": path,
+                "data": data
+            }
+
+            res = CreateDbfsBlockOp(
+                name="createdbfsitem",
+                block_name=block_name,
+                path=path,
+                data=data
+            )
+            
+            self.assert_res(res, expected_spec)
+
+        kfp.compiler.Compiler()._compile(my_pipeline)
```

Comment:
`_compile()` is being deprecated, maybe switch to `_create_workflow()`?

Reply:
Done